app/routes/messaging.py

```diff
@@ -19,15 +19,21 @@ async def send_messages(
 ):
     """
     Send basic message.
```

Comment:
Just a pedantic request, for consistency -- headings don't need to end in punctuation